lock-facebook/src/main/java/com/auth0/android/facebook/FacebookSignInDelegateActivity.java

```diff
@@ -0,0 +1,35 @@
+package com.auth0.android.facebook;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+
+import com.facebook.login.LoginManager;
+
+import java.util.Collections;
+import java.util.List;
+
+public class FacebookSignInDelegateActivity extends Activity {
+
+    public static final String FACEBOOK_PERMISSIONS_EXTRA = "com.auth0.android.facebook.FACEBOOK_PERMISSIONS";
```

Comment:
these can be package local instead of public? the same with the activity itself?